types.go

```diff
@@ -260,14 +260,14 @@ type CheckResult struct {
 }
 
 // ApiCheckResult represents an API Check result
-type ApiCheckResult struct {
+type ApiCheckResult map[string]interface {
```

Comment:
Just for clarity this should be a one-liner `type ApiCheckResult map[string]interface {}`.. struct was supposed to have fields inside, but interface type can't

Reply:
Sure, I can delete that quickly